code/lib/cli/src/generators/baseGenerator.ts

```diff
@@ -103,12 +97,16 @@ const stripVersions = (addons: string[]) => addons.map((addon) => getPackageDeta
 const hasInteractiveStories = (rendererId: SupportedRenderers) =>
   ['react', 'angular', 'preact', 'svelte', 'vue', 'vue3', 'html'].includes(rendererId);
 
+const hasFrameworkTemplates = (framework?: SupportedFrameworks) =>
```

Comment:
We needed a way to identify the frameworks with framework-specific template files.